batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/controller/UserController.java

```diff
@@ -0,0 +1,170 @@
+package com.lftechnology.batch7crud.controller;
+
+import java.awt.Window.Type;
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.service.UserService;
+import com.lftechnology.batch7crud.util.TypeCaster;
+
+/**
+ * @author madandhungana <madandhungana@lftechnology.com> Jan 18, 2016
+ */
+
+@WebServlet("/users/*")
+public class UserController extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    private UserService userService = new UserService();
+
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public UserController() {
+        super();
+        // TODO Auto-generated constructor stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)